frontend/src/lib/utils.tsx

```diff
@@ -1137,3 +1137,9 @@ export function ensureStringIsNotBlank(s?: string | null): string | null {
 export function setPageTitle(title: string): void {
     document.title = title ? `${title} • PostHog` : 'PostHog'
 }
+
+export function isSingleSeries(formula?: string): boolean {
+    if (!formula) return false
```

Comment:
If it's not got a formula, does that mean it's not a single series? In Trends can't you just select one "action" and have it plot as a single series?

Reply:
Maybe we can keep close to what we're deciding on here, if we should get persons for a specific series. `isValidToRetrievePersons`. `isSingleSeries` sounds very abstract for a util also. I would, until it looks like it's happening a lot, keep this local to the ActionsLineGraph

non-blocking